core/Cargo.toml

```diff
@@ -21,6 +21,7 @@ lazy_static = "1.4.0"
 log = "0.4.21"
 ollama-rs = "=0.2.1"
 pgmq = "0.29"
+pgrx = "=0.12.5"
```

Comment:
remove pgrx from `./core/Cargo.toml`. Core is separate so we can have a library that has no dependency on pgrx or any postgres service side.

Reply:
for PgOid we need it if we support

Reply:
the only place PgOid type would be used is on the `vectorize.table()` outer most function, in the signature and then using it to determine `table` and `schema`.